CHANGELOG.md

```diff
@@ -4,8 +4,19 @@ All notable changes to this project will be documented in this file.
 
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/), and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## [0.2.0]
 
-## [0.1.2] - Unreleased
+[0.2.0]: https://github.com/streem/streem-sdk-jvm/compare/v0.2.0...HEAD
+
+### Added
+
+* Support for a `reservation_sid` claim, if using the Group Reservation feature. 
+
+### Changed
+
+### Fixed
+
+## [0.1.2]
```

Comment:
0.1.2 was unreleased. That was meant to be a placeholder for the next version. So you can just replace 0.1.2 with 0.2.0.

Reply:
Also please update the version in `gradle.properties` to `0.2.0-SNAPSHOT` since you're bumping the version.

Reply:
Done 👍